### PR TITLE
Click tracking

### PR DESCRIPTION
**Pull request by matthewwrobel**
-Implemented ClickWrapper HOC to log all user click metadata to atelier-api
-Wrapped all modules with click-tracking functionality

@jabrake @tchitrakorn @tchitrakorn no rush on this, thanks!

### CODE REVIEW

File: client/ClickWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const ClickWrapper = (ComponentToWrap, moduleName) => {
+  return (props) => (
+    <div onClick={(e) => {
+      console.log('clicked element:', e.target.outerHTML, '\nclicked module', moduleName, '\ntimestamp:', new Date);
```
Comment (jabrake) [inline client/ClickWrapper.jsx]: No rush but we should remove this console.log in a future PR!

File: client/ratings_reviews/RatingsReviews.jsx [hunk 2]
```diff
@@ -119,7 +119,7 @@ class RatingsReviews extends React.Component {
   render() {
     if (!$.isEmptyObject(this.state.meta) && this.props.info !== null) {
       return (
-        <div class='review-overall-container'>
+        <div id="ratings-and-reviews" class='review-overall-container'>
```
Comment (jabrake) [inline client/ratings_reviews/RatingsReviews.jsx]: Is this something you added for Tatt's component?

**Review by jabrake — APPROVED** [review]
This was pretty quick to review since you showed us how it all works :) Just a couple of questions but looks good to merge!

File: client/ratings_reviews/RatingsReviews.jsx (hunk 2 above)
Comment (matthewwrobel) [reply client/ratings_reviews/RatingsReviews.jsx]: Ah oops. Should have taken that out. My first idea was to find the module names by recursively checking all parent elements for IDs. I'll remove that.

File: client/ClickWrapper.jsx (hunk 1 above)
Comment (matthewwrobel) [reply client/ClickWrapper.jsx]: Totes, will do.

**Pull request merged**